GithubApiWithVIPER/GithubApiWithVIPERTests/GithubApiWithVIPERTests.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  GithubApiWithVIPERTests.swift
+//  GithubApiWithVIPERTests
+//
+//  Created by Yusuf Özgül on 19.09.2020.
+//  Copyright © 2020 Yusuf Özgül. All rights reserved.
+//
+
+import XCTest
+@testable import GithubApiWithVIPER
+
+/// test cases for repo search module
+class GithubApiWithVIPERTSearchModuleests: XCTestCase {
+    private var presenter: GithubSearchPresenter!
+    private var interactor: GithubSearchInteractor!
+    private var view: MockView!
+    private var router: MockRouter!
+    
+    override func setUp() {
+        interactor = GithubSearchInteractor()
```

Comment:
interactor'ın da mock'unu oluştursan güzel olur yusuf ^_^

Reply:
sonradan gördüm de **searchBarTextDidChange** bunun testini yapabilmek için gerekli mockInteractor